src/App.css

```diff
@@ -1,38 +1,14 @@
 .App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
   min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
+  background-color: #282c34;
```

Comment:
Is there a reason for this colour not to be in a variable? It might be good to have this colour in a separate variable

Reply:
we've discussed this offline, I'm happy with the changes!!
Since we're not using sass, this approach is preferable.